Changes

```diff
@@ -54,6 +54,15 @@ Working version
 
 ### Other libraries:
 
+* #9071, #9100, #10935: Reimplement `Thread.exit()` as raising the
+  exception `Thread.Exit`, and mark `Thread.exit` as deprecated.
+  The new implementation changes the behavior compared with
+  the previous implementation.  For example, the new implementation
+  causes `Fun.finally` finalizers to be run and catch-all exception
+  handlers to abort termination.
+  (Jacques-Henri Jourdan and Xavier Leroy, review by Damien Doligez,
+  Guillaume Munch-Maccagnoni, and Enguerrand Decorne)
```

Comment:
Let's credit Gabriel who contributed to the discussion.

First mention that the main addition is the new exception `Thread.Exit`?

Reply:
My cunning plan is to mention the addition of `Thread.Exit` in the 4.14 section of Changes.  But it means  I must sneak it into 4.14 first before the Changes file can be completed.